debian/rules

```diff
@@ -21,7 +21,10 @@ CONFIGURE_FLAGS := \
 	--enable-systemd \
 	--enable-threads \
 	--enable-sixaxis \
-	--enable-experimental
+	--enable-experimental \
+	--enable-deprecated \
+	--enable-testing \
+	--enable-phonebook-ebook
```

Comment:
This was removed here https://github.com/ubports/bluez-packaging/commit/fed7a89a7514612eb155ca1c9635faf140a5409d so i dont think we need it?

Reply:
Ouch yes will remove it